backends/bmv2/simpleSwitch.cpp

```diff
@@ -206,7 +207,25 @@ SimpleSwitch::convertExternObjects(Util::JsonArray *result,
             // Do not generate any code for this operation
         }
     } else {
-        error("Unknown extern type %1%", em->originalExternType->name);
+        if (emitExterns) {
+            ::warning("Found extern method %1%. Make sure that the BMv2 supports it",
```

Comment:
I would think that this warning is unnecessary - if the user has passed the suitable flag to the compiler they already should know this.